src/alert/alert.scss

```diff
@@ -17,43 +18,53 @@
     background-color: t(iui-color-background-1);
   }
 
-  &-icon {
-    @include iui-icons-default;
-    flex-shrink: 0;
-    margin-left: $iui-m;
-  }
-
-  &-message {
-    margin: $iui-baseline $iui-m;
-  }
-
-  &-link {
-    @include iui-anchor-underline('on-initial');
-    border-radius: $iui-border-radius;
-    cursor: pointer;
-    margin-left: $iui-s;
-    user-select: none;
-    white-space: nowrap;
-  }
-
-  > .iui-button {
-    margin-left: auto;
-    margin-right: $iui-s;
-  }
-
   &.iui-sticky {
```

Comment:
I think it makes sense to make this into it's own mixin and then include it in `classes.scss` like this:
```scss
.iui-alert {
  @include iui-alert;

  &.iui-sticky {
    @include iui-alert-sticky;
  }
}
```
I ran into a similar issue in #568 with the anchor mixin.  That mixin included the `.iui-anchor-external` modifier which I did not need in breadcrumbs & was causing the specificity to go high.